pom.xml

```diff
@@ -336,10 +336,88 @@
 					</execution>
 				</executions>
 			</plugin>
+
+			<plugin>
+				<groupId>org.graalvm.buildtools</groupId>
+				<artifactId>native-maven-plugin</artifactId>
+				<version>0.9.20</version>
```

Comment:
How about we store this version in a property? Looks like we use it in a couple of places in this file.

Reply:
Done! 
Also removed explicit declare of `spring-boot-maven-plugin` version as it's inherited from spring-boot-dependencies and keeps in sync with spring boot versions. (ref [here](https://github.com/spring-projects/spring-boot/blob/a9ff0f0ca3aa3f5f39c57097a3ccc6ab6c44d458/spring-boot-project/spring-boot-dependencies/build.gradle#L1308))
As for `junit-platform-launcher` version, kept as is because according to [doc](https://graalvm.github.io/native-build-tools/latest/maven-plugin.html#testing-support-version-compatibility), we should be able to remove it after we upgrade Maven Surefire plugin to 3.1.2 (#1936)